pkgs/tools/security/pcsc-safenet/default.nix

```diff
@@ -1,22 +1,31 @@
 { stdenv
 , lib
+, runCommand
 , fetchurl
 , autoPatchelfHook
 , dpkg
-, gtk2
-, openssl
+, gtk3
+, openssl_1_1
 , pcsclite
+, unzip
 }:
 
 stdenv.mkDerivation rec {
   pname = "pcsc-safenet";
-  version = "10.0.37-0";
-
-  # https://aur.archlinux.org/packages/sac-core/
-  src = fetchurl {
-    url = "https://storage.spidlas.cz/public/soft/safenet/SafenetAuthenticationClient-core-${version}_amd64.deb";
-    sha256 = "1r9739bhal7ramj1rpawaqvik45xbs1c756l1da96din638gzy5l";
-  };
+  version = "10.8.28";
+
+  # extract debian package from larger zip file
+  src = runCommand "sac.deb" {
```

Comment:
Why `fetchzip` is not enough here?

Reply:
The only argument against fetchzip is that this way we only unpack the one file we need, and not the others we don't. If the extra size of the intermediate build outputs isn't too much of a cost to pay for standardization, I'll make that change.

Reply:
Yes, I think, we already have the ZIP, so it does not cost us more to do a fetchzip.